README.md

```diff
@@ -1 +1,194 @@
-﻿
\ No newline at end of file
+﻿# Overview
+
+This NApp integrates Kafka with the Kytos SDN platform to enable event-driven messaging and real-time event streaming.
+
+# Planned Features
+
+- Asynchronous Kafka producer with support for compression and acknowledgments.
+- Event listener for new KytosEvents, which then serializes and publishes events to Kafka.
+- Resilient Kafka client with automatic retries for connectivity issues.
+- Uses Main asyncio loop to handle asynchronous tasks serialization and publishing.
+- Regex filtering logic to handle serialization permission easily and efficiently.
+- Endpoints to dynamically add, list, and remove serialization permissions.
+
+# Requirements
+
+- [aiokafka](https://aiokafka.readthedocs.io/en/stable/)
+
+# Events
+
+## Subscribed
+
+- All core NApps (`kytos/*` and `kytos.*`)
+    - `kytos/mef_eline.*`
+    - `kytos/of_core.*`
+    - `kytos/flow_manager.*`
+    - `kytos/topology.*`
+    - `kytos/of_lldp.*`
+    - `kytos/pathfinder.*`
+    - `kytos/maintenance.*`
+
+# Filtering
+
+Event consumption and serialization follows the principle of `least privilege`, meaning events must be explicitly accepted to be allowed to be propagated to Kafka. Filtering logic uses `regex` to quickly accept or deny incoming events, based on preset patterns. Currently, the NApp mainly supports wildcard logic, but can be easily extended for matching as well:
+
+## Wildcard
+
+The expected functionality, takes in any regex logic and compares values to them
+
+```
+# Example
+{"pattern": "kytos[./](.*)", "description": "Allows all core NApps"}
+```
+
+## Match
+
+To achieve match functionality, you must start and end your match with `^` and `$` respectively.
+
+```
+# Example
+{"pattern": "^amlight/pathfinder.created$", "description": "Allow ONLY this pattern"}
+```
+
+# Planned Endpoint
+
+## GET /v1/filters
+
+Lists the summary of all Filter objects in the filtering pipeline, where the dynamic key is the UUID of the Filter. The response looks similar to the following:
+
+```
+[
+    "AOIJ8A9JCA898XA01": {
+        "pattern": str,
+        "mutable": bool,
+        "description": str
+    },
+    "OPKAS9C0A2OJE180U": {
+        "pattern": str,
+        ...
+    },
+    ...
+]
+```
+
+# Development
```

Comment:
I'd recommend you nuking the development info here and cross reference this [setup dev env doc we have](https://github.com/kytos-ng/documentation/blob/master/tutorials/napps/development_environment_setup.rst#setting-up-your-dev-environment) instead. We centralized it there to avoid repetition in multiple readmes and since the installation of all NApps are supposed to be the same, then it's less documentation house keeping to keep up with too. 

We used to have this on NApps readmes but then certain fixes or upgrades in Python version, or frozen pip related stuff it'd require 10+ PRs easily making code maintenance difficult for us.